airflow/api_fastapi/execution_api/datamodels/xcom.py

```diff
@@ -26,5 +26,5 @@ class XComResponse(BaseModel):
     """XCom schema for responses with fields that are needed for Runtime."""
 
     key: str
-    value: Any
+    value: JsonValue
```

Comment:
Re-generate datamodels too to update the type:

https://github.com/apache/airflow/blob/8f63b828ace09b1095229f22b0c6d1f0f85ea81a/task_sdk/src/airflow/sdk/api/datamodels/_generated.py#L244-L250

Reply:
Ah good catch.